charts/gardener-extension-provider-alicloud/values.yaml

```diff
@@ -26,6 +26,9 @@ controllers:
     concurrentSyncs: 5
   dnsrecord:
     concurrentSyncs: 5
+    providerClientQPS: 25
```

Comment:
Would you mind provide a link where AliCloud specify the DNS Api ratelimit and put it above `providerClientQPS` as a comment?

Reply:
Unfortunately there is no official document for it.  Ali supports said maximum QPS is 60